.github/main.workflow

```diff
@@ -0,0 +1,8 @@
+workflow "New workflow" {
+  on = "push"
+  resolves = ["Find Python 3 syntax errors and undefined names"]
+}
+
+action "Find Python 3 syntax errors and undefined names" {
+  uses = "cclauss/Find-Python-syntax-errors-action@master"
```

Comment:
What does this reference?  Is it a new file you forgot to `git add`?

Reply:
It references https://github.com/cclauss/Find-Python-syntax-errors-action but I think that you need to be in the GitHub Actions beta program for this to work.  It works great on my repos but unfortunately I don't think it will work on yours until GitHub Actions graduates from beta state.

Reply:
Ok, in that case is there another approach we should try? Or should we close this PR and reopen when it’s GA?